qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -150,10 +150,10 @@ def decompose_clifford_ag(clifford):
         _set_row_z_zero(clifford_cpy, circuit, i)
 
     for i in range(clifford.num_qubits):
-        if clifford_cpy.destabilizer.phase[i]:
+        if clifford_cpy.phase[i]:  # destabilizer
             _append_z(clifford_cpy, i)
             circuit.z(i)
-        if clifford_cpy.stabilizer.phase[i]:
+        if clifford_cpy.phase[i + clifford.num_qubits]:  # stablizer
```

Comment:
Is this a sign that we should have `stab_phase` and `destab_phase` shortcuts as you've already added `stab_x` (`stab_z`) and `destab_x` (`desstab_z`)?

Reply:
Thank you for great suggestion.
Added in 
bec74a6
3c9e9bc